authors/apps/articles/urls.py

```diff
@@ -1,7 +1,9 @@
 from django.urls import path
 from . import views
-
+from authors.apps.comments.views import CommentView,CommentDetailView
 urlpatterns = [
     path('', views.ArticleListCreateView.as_view(), name='article-create'),
-    path('<slug>/', views.ArticleUpdateDeleteView.as_view(), name='article-update')
+    path('<slug>/', views.ArticleUpdateDeleteView.as_view(), name='article-update'),
+    path('<slug>/comments/',CommentView.as_view(),name='comments'),
```

Comment:
@MuhweziDeo Move the urls to the comments app. remember the app has to stand alone.